study-builder/studies/rarex/i18n/en.json

```diff
@@ -57,16 +57,24 @@
   "consent_summary_created": "Please sign the Research Consent Form, which will allow RARE-X to obtain biospecimen samples, copies of your medical records, and opt-in to share blood samples and/or other biosamples.",
   "consent_summary_in_progress": "Please finish reviewing and signing the Research Consent Form.",
   "consent_summary_complete": "Thank you for signing the Research Consent Form.",
-  "consent_signature_first_name": "Your complete legal first (given) name*",
+  "consent_signature_first_name": "Your complete legal first (given) name *",
   "consent_signature_first_name_validation_hint": "First name cannot be empty",
   "consent_signature_first_name_exp": "Your complete legal first (given) name",
-  "consent_signature_last_name": "Your complete legal last (family) name*",
+  "consent_signature_last_name": "Your complete legal last (family) name *",
   "consent_signature_last_name_validation_hint": "Last name cannot be empty",
   "consent_signature_last_name_exp": "Your complete legal last (family) name",
   "consent_signature_middle_name": "Your middle/second or additional (if you have one) name",
   "consent_signature_middle_name_exp": "Your complete middle/second or additional name",
   "consent_signature_date_signed": "Date Signed",
 
+  "lar_consent_child_firstname": "Participant's complete legal given (first) name *",
+  "lar_consent_child_firstname_validation_hint": "Participant's first name cannot be empty",
+  "lar_consent_child_firstname_exp": "Participant’s complete legal given (first) name at birth",
+  "lar_consent_child_lastname": "Participant’s complete legal family (last) name *",
+  "lar_consent_child_lastname_validation_hint": "Participant's last name cannot be empty",
+  "lar_consent_child_lastname_exp": "Participant’s complete legal family (last) name at birth",
+  "lar_consent_child_middlename": "Participant's middle/second or additional (if your child has one) name",
+  "lar_consent_child_middlename_exp": "Participant’s complete middle/second or additional name",
```

Comment:
Bad merge. Looks like these are duplicates. The ones that say `Patient's` are the correct ones.